### ISSUE TITLE
[ERROR] DeepLabV3+DenseNet Backbone

### ISSUE THREAD

**Issue by stigma0617** [donnyyou/torchcv#44]
Hi, 

I have a question when I try to train DeepLabV3 with DenseNet backbone.

[the loss term](https://github.com/donnyyou/torchcv/blob/master/scripts/seg/cityscapes/run_fs_deeplabv3_cityscapes_seg.sh#L20) of DeepLabV3.

I wonder why you use this auxiliary loss term. As far as I know, original DeepLabV3 was not trained by the auxiliary cross-entropy loss.

As your [code ](https://github.com/donnyyou/torchcv/blob/master/models/seg/nets/deeplabv3.py#L70) about deeplabv3 was hard-coded, it should be fixed for other backbone (i.g., DenseNet). 

Thanks

**Comment by donnyyou** [comment]
1. Original DeepLabV3 maybe not be trained with auxiliary, and I haven't tested the influence of it. 
2. The code is really hard-coded. You could simply fix it by replacing 2048 with self.backbone.num_features. 
3. Welcome your pull request!!!
